### PR TITLE
FEAT: support baichuan-chat pytorch model

### PR DESCRIPTION
**Pull request by pangyoki**
ATT

### CODE REVIEW

File: xinference/model/llm/pytorch/core.py [hunk 1]
```diff
@@ -224,6 +232,8 @@ def chat(
         full_prompt = self._to_prompt(prompt, system_prompt, chat_history=chat_history)
 
         generate_config = self._sanitize_generate_config(generate_config)
+        if "stop" not in generate_config and self._stop is not None:
```
Comment (UranusSeven) [inline xinference/model/llm/pytorch/core.py:235]: This should be part of `_sanitize_generate_config`

Comment (pangyoki) [reply xinference/model/llm/pytorch/core.py]: done

File: xinference/model/llm/pytorch/core.py [hunk 2]
```diff
@@ -204,13 +212,27 @@ def __init__(
         sep: str,
         user_name: str,
         assistant_name: str,
+        stop: Optional[str] = None,
```
Comment (UranusSeven) [inline xinference/model/llm/pytorch/core.py:215]: Could be a list.

Comment (pangyoki) [reply xinference/model/llm/pytorch/core.py]: done

Comment (pangyoki) [reply xinference/model/llm/pytorch/core.py]: done

**Review by UranusSeven — APPROVED** [review]

**Pull request merged**